app/assets/javascripts/discourse/app/components/sidebar/section.js

```diff
@@ -39,6 +39,11 @@ export default class SidebarSection extends Component {
     } else {
       this.keyValueStore.setItem(this.collapsedSidebarSectionKey, true);
     }
+
+    // remove focus from the toggle, but only on click
+    if (!event.key) {
+      document.activeElement.blur();
```

Comment:
I'm a little uncomfortable with this, even though it is limited to click events. It feels like a styling issue, as in, we should be able to handle it using just CSS. Maybe this is a use case for `:focus-visible`?

Reply:
Yeah, I didn't mean to trigger a whole refactor here... 

It is also odd to have a button as the element for this toggle. We could use a non-button element, like a simple `div` (yes screen readers will complain, but given this is visual candy, it might be ok). We've also used `details/summary` elements in the advanced search page for similar UI, that's another option, not sure how styleable those elements are for the sidebar. 

Or, a styling change as a quick fix: have the section buttons/links not need a background style on their container on hover/focus so that we don't have to use `:focus-within`...